extensions/lifecycle/conductor/LifecycleConductor.js

```diff
@@ -92,6 +101,48 @@ class LifecycleConductor {
         }]));
     }
 
+    _processBucketWithAccountId(ownerId, bucketName, done) {
+        this.logger.debug('processing bucket', {
+            method: 'LifecycleConductor::_processBucketWithAccountId',
+            ownerId,
+            bucketName,
+        });
+        const client = this._vaultClientCache.getClient(LIFEYCLE_CONDUCTOR_CLIENT_ID);
+        const opts = {};
+        return client.getAccountIds([ownerId], opts, (err, res) => {
+            if (err) {
+                this.logger.error(
+                    'failed to retrieve bucket owner account id, skipping', {
+                        error: err,
+                        bucket: bucketName,
+                        canonicalId: ownerId,
+                    });
+                return process.nextTick(done);
```

Comment:
`process.nextTick` not needed since `_processBucketWithAccountId` is defined to be asynchronous and operates asynchronously.